tests/bazel_tests.bzl

```diff
@@ -204,10 +198,12 @@ def _bazel_test_script_impl(ctx):
             [workspace_file, build_file] + ctx.files.extra_files,
             collect_data = True,
         ),
+        executable = script_file,
     )
 
-_bazel_test_script = go_rule(
+_bazel_test_script_test = go_rule(
```

Comment:
Please don't rename this or set `test = True` or return `executable = script_file` above. This is a rule that produces a script that may be run in a test. It is not a test itself because it doesn't produce an executable.

Sorry for the annoying distinction: on Windows, files can't be marked executable, and shell scripts by themselves are not executable. This is why we need to wrap the `native.sh_test` around the generated script in the `bazel_test` macro below.